test/es-module/test-esm-detect-ambiguous.mjs

```diff
@@ -393,3 +393,18 @@ describe('Wrapping a `require` of an ES module while using `--abort-on-uncaught-
     strictEqual(signal, null);
   });
 });
+
+describe('when working with Worker threads', () => {
+  it('should work', async () => {
```

Comment:
What would a more precise description be? This?

```suggestion
  it('should evaluate a CommonJS worker as valid sloppy script where the CommonJS scope variables do not exist', async () => {
```

I read https://nodejs.org/api/worker_threads.html#new-workerfilename-options and I don’t see a mention of these variables intentionally being missing; is this documented somewhere? If I pass a filename, the variables such as `require` exist in there, right? It feels like a bug that this is being parsed as CommonJS but the wrapper variables aren’t getting passed in.